test/lib/smart-survey-api.test.js

```diff
@@ -8,15 +8,18 @@ let getResponsesStub = sinon.stub();
 const smartSurveyStub = function() {
     return {getResponses: getResponsesStub};
 };
-// Proxyquire syntax: first argument: point to your module (which holds your dependency)
-// 2nd argument: An object with the key: module name to proxy & the value the object above
+// Proxyquire syntax:
+// first argument: points to your module (which holds your dependency)
+// 2nd argument: An object. The object has a key that is the name of the moduleof the dependency
+// & the value the object above of the proxy
 // So this means smartSurveyStub will now point to this function, i.e. a stub
-const smartSurveyAPI = proxyquire('../../lib/smart-survey-api', { 'smartsurvey-client': smartSurveyStub});
+const SmartSurveyAPIBase = proxyquire('../../lib/smart-survey-api', { 'smartsurvey-client': smartSurveyStub});
+const smartSurveyAPI = new SmartSurveyAPIBase('mytoken', 'mytokensecret');
```

Comment:
Classes are for object-orientation and in my book object-orientation is better replaced by pure functional programming. But that's just my opinion which I why I was asking what classes are giving you?

Do they make things easier in any way or are you just using them because they are ES6?